deployments/warp_routes/tETH/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "Turbo ETH",
+  "symbol": "tETH",
+  "description": "Warp Route tETH",
+  "image": "",
```

Comment:
no image?

Reply:
couldn't find any logo, just asked them about it. The warp UI can be updated when one becomes available luckily